From 57bfdcafd87a8b8aafb7291c640c838f1b987194 Mon Sep 17 00:00:00 2001
From: nick black <nick.black@sprezzatech.com>
Date: Fri, 14 Dec 2012 00:09:58 -0500
Subject: [PATCH 3/4] check for kernel videodev, not libv4l

---
 configure.ac | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index d905318..33f30c5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -917,9 +917,7 @@ AC_ARG_WITH(libv4l, [  --without-libv4l        build without V4L support])
 
 have_v4l="no"
 if test "x$with_libv4l" != "xno"; then
-  PKG_CHECK_MODULES(V4L, libv4l2,
-    have_v4l="yes",
-    have_v4l="no  (v4l library not found)")
+  AC_CHECK_HEADER(linux/videodev2.h, have_v4l="yes")
 fi
 
 AM_CONDITIONAL(HAVE_V4L, test "$have_v4l" = "yes")
-- 
1.8.5.3

