--- gegl-0.3.0/operations/external/Makefile.am~	2014-02-21 03:12:40.000000000 +0100
+++ gegl-0.3.0/operations/external/Makefile.am	2014-02-21 18:33:47.639453783 +0100
@@ -114,7 +114,7 @@ endif
 if HAVE_UMFPACK
 ops += matting-levin.la
 matting_levin_la_SOURCES = matting-levin.c matting-levin-cblas.c matting-levin-cblas.h
-matting_levin_la_LIBADD  = $(op_libs) $(UMFPACK_LIBS)
+matting_levin_la_LIBADD  = $(op_libs) $(UMFPACK_LIBS) -lsuitesparseconfig
 matting_levin_la_CFLAGS  = $(AM_CFLAGS)
 endif
 
