--- ./operations/external/webp-load.c~	2016-02-12 15:05:43.000000000 +0000
+++ ./operations/external/webp-load.c	2016-03-14 00:49:53.252723683 +0000
@@ -100,7 +100,7 @@
                                      NULL, &error);
   if (!success || error != NULL)
     {
-      g_warning (error->message);
+      g_warning ("%s", error->message);
       g_error_free (error);
       return -1;
     }
@@ -130,7 +130,7 @@
                                          NULL, &error);
       if (!success || error != NULL)
         {
-          g_warning (error->message);
+          g_warning ("%s", error->message);
           g_error_free (error);
           return -1;
         }
@@ -208,7 +208,7 @@
       p->stream = gegl_gio_open_input_stream (o->uri, o->path, &p->file, &error);
       if (p->stream == NULL)
         {
-          g_warning (error->message);
+          g_warning ("%s", error->message);
           g_error_free (error);
           cleanup (operation);
           return;
--- ./operations/external/tiff-load.c.orig	2016-02-23 21:52:49.000000000 +0000
+++ ./operations/external/tiff-load.c	2016-03-14 01:02:15.822522210 +0000
@@ -119,7 +119,7 @@
   gchar *message;
 
   g_vasprintf(&message, format, arguments);
-  g_warning(message);
+  g_warning("%s", message);
 
   g_free(message);
 }
@@ -132,7 +132,7 @@
   gchar *message;
 
   g_vasprintf(&message, format, arguments);
-  g_message(message);
+  g_message("%s", message);
 
   g_free(message);
 }
@@ -158,7 +158,7 @@
                                  NULL, &error);
       if (read < 0)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
     }
@@ -189,7 +189,7 @@
                                          NULL, &error);
               if (read < 0)
                 {
-                  g_warning(error->message);
+                  g_warning("%s", error->message);
                   g_error_free(error);
                   break;
                 }
@@ -242,7 +242,7 @@
         position = g_seekable_tell(G_SEEKABLE(p->stream));
       else
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
     }
@@ -284,7 +284,7 @@
                                 NULL, &error);
   if (!closed)
     {
-      g_warning(error->message);
+      g_warning("%s", error->message);
       g_error_free(error);
     }
 
@@ -322,7 +322,7 @@
                                NULL, &error);
       if (info == NULL)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
       else
@@ -728,7 +728,7 @@
         p->can_seek = g_seekable_can_seek(G_SEEKABLE(p->stream));
       if (p->stream == NULL)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
           cleanup(operation);
           return;
--- ./operations/external/svg-load.c~	2016-02-12 15:06:15.000000000 +0000
+++ ./operations/external/svg-load.c	2016-03-14 09:31:31.337260850 +0000
@@ -164,7 +164,7 @@
       stream = gegl_gio_open_input_stream (o->uri, o->path, &p->file, &error);
       if (stream == NULL)
         {
-          g_warning (error->message);
+          g_warning ("%s", error->message);
           g_error_free (error);
           cleanup (operation);
           return;
@@ -175,7 +175,7 @@
                                                     NULL, &error);
       if (p->handle == NULL)
         {
-          g_warning (error->message);
+          g_warning ("%s", error->message);
           g_error_free (error);
           cleanup (operation);
           return;
--- ./operations/external/jp2-load.c~	2016-02-12 15:04:50.000000000 +0000
+++ ./operations/external/jp2-load.c	2016-03-14 09:35:33.713735532 +0000
@@ -88,7 +88,7 @@
                                  NULL, &error);
       if (read < 0)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
           break;
         }
@@ -265,7 +265,7 @@
       stream = gegl_gio_open_input_stream(o->uri, o->path, &p->file, &error);
       if (stream == NULL)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
           cleanup(operation);
           return;
--- ./operations/external/tiff-save.c.orig	2016-02-23 21:52:49.000000000 +0000
+++ ./operations/external/tiff-save.c	2016-03-14 09:46:52.221858319 +0000
@@ -94,7 +94,7 @@
   gchar *message;
 
   g_vasprintf(&message, format, arguments);
-  g_warning(message);
+  g_warning("%s", message);
 
   g_free(message);
 }
@@ -107,7 +107,7 @@
   gchar *message;
 
   g_vasprintf(&message, format, arguments);
-  g_message(message);
+  g_message("%s", message);
 
   g_free(message);
 }
@@ -144,7 +144,7 @@
                                       NULL, &error);
       if (written < 0)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
     }
@@ -192,7 +192,7 @@
         position = g_seekable_tell(G_SEEKABLE(p->stream));
       else
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
     }
@@ -243,7 +243,7 @@
                                           NULL, &error);
           if (written < 0)
             {
-                  g_warning(error->message);
+                  g_warning("%s", error->message);
                   g_error_free(error);
                   break;
             }
@@ -256,7 +256,7 @@
                                  NULL, &error);
   if (!closed)
     {
-      g_warning(error->message);
+      g_warning("%s", error->message);
       g_error_free(error);
     }
 
@@ -293,7 +293,7 @@
                                NULL, &error);
       if (info == NULL)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
       else
@@ -567,7 +567,7 @@
   if (p->stream == NULL)
     {
       status = FALSE;
-      g_warning(error->message);
+      g_warning("%s", error->message);
       goto cleanup;
     }
 
